utils/gcp/object_storage.cc

```diff
@@ -157,12 +157,12 @@ class utils::gcp::storage::client::object_data_sink : public data_sink_impl {
                     auto& buf = bufs.back();
                     if (buf.size() > rem) {
                         auto keep = buf.size() - rem;
-                        _buffers.emplace_back(buf.share(keep, rem));
+                        _buffers.emplace(_buffers.begin(), buf.share(keep, rem));
```

Comment:
https://en.cppreference.com/w/cpp/container/deque/emplace_front.html
```suggestion
                        _buffers.emplace_front(buf.share(keep, rem));
```